system-contracts/contracts/Constants.sol

```diff
@@ -92,12 +97,14 @@ enum SystemLogKey {
     PREV_BATCH_HASH_KEY,
     CHAINED_PRIORITY_TXN_HASH_KEY,
     NUMBER_OF_LAYER_1_TXS_KEY,
+    BLOB_ONE_HASH_KEY,
+    BLOB_TWO_HASH_KEY,
```

Comment:
correct, given that these 2 logs are mandatory i figured they should come before the system contract upgrade. we can change this so that we only append values instead of modifying but i don't see much downside

Reply:
cc @StanislavBreadless @vladbochok in case you guys feel strongly here as well

Reply:
It is not a breaking as the new L1 contracts will process only the new transactions (and the old L1 contracts will process only the old system contracts). Aesthetically, I like @koloz193's approach, but I don't have any strong opinions either

Reply:
It is ok imho.